Mathlib/Data/Matrix/Mul.lean

```diff
@@ -692,17 +687,31 @@ theorem add_vecMul [Fintype m] (A : Matrix m n α) (x y : m → α) :
   ext
   apply add_dotProduct
 
-theorem vecMul_smul [Fintype n] [NonUnitalNonAssocSemiring S] [DistribSMul R S]
-    [IsScalarTower R S S] (M : Matrix n m S) (b : R) (v : n → S) :
-    (b • v) ᵥ* M = b • v ᵥ* M := by
-  ext i
-  simp only [vecMul, dotProduct, Finset.smul_sum, Pi.smul_apply, smul_mul_assoc]
-
 theorem mulVec_smul [Fintype n] [NonUnitalNonAssocSemiring S] [DistribSMul R S]
     [SMulCommClass R S S] (M : Matrix m n S) (b : R) (v : n → S) :
     M *ᵥ (b • v) = b • M *ᵥ v := by
-  ext i
-  simp only [mulVec, dotProduct, Finset.smul_sum, Pi.smul_apply, mul_smul_comm]
+  ext
```

Comment:
Minor nit: looks like the existing code always has `Matrix m n`, but asks for either `Fintype m` or `Fintype n` as appropriate, while the new code always has `Fintype n` but switches back and forth between `Matrix m n` and `Matrix n m`. I think the previously-used approach is better for readability. (Also, if we have the same typeclass assumption in 4 consecuitive lemmas we should probably make it a section-level variable.)

Reply:
Well spotted, indeed we should stick to `Matrix m n` everywhere